src/storm/solver/IterativeMinMaxLinearEquationSolver.h

```diff
@@ -1,6 +1,8 @@
 #pragma once
 
+#include <variant>
```

Comment:
Do we need this include here? It is a standard header, so we can maybe add it to the PCH, but still?